pyqtgraph/functions.py

```diff
@@ -75,9 +75,8 @@ def siScale(x, minVal=1e-25, allowUnicode=True):
             pref = SI_PREFIXES[m+8]
         else:
             pref = SI_PREFIXES_ASCII[m+8]
-    p = .001**m
-    
-    return (p, pref)    
+    m1 = -3*m
+    p = float('1E{0}'.format(m1))
```

Comment:
Somewhat cleaner is
```suggestion
    p = 10.**m1
```
, as it abstains from any type casting and formatting, while ensuring the same amount of precision.